cartography/intel/semgrep/findings.py

```diff
@@ -0,0 +1,217 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import neo4j
+import requests
+
+from cartography.client.core.tx import load
+from cartography.graph.job import GraphJob
+from cartography.models.semgrep.findings import SemgrepDeploymentSchema
+from cartography.models.semgrep.findings import SemgrepSCAFindingSchema
+from cartography.models.semgrep.findings import SemgrepSCALocationSchema
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+def get_deployment(semgrep_app_token: str) -> List[Dict[str, Any]]:
+    """
+    Gets the deployment associated with the passed Semgrep App token.
+    param: semgrep_app_token: The Semgrep App token to use for authentication.
+    """
+    deployment = {}
+    deployment_url = "https://semgrep.dev/api/v1/deployments"
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": f"Bearer {semgrep_app_token}",
+    }
+    response = requests.get(deployment_url, headers=headers)
+    response.raise_for_status()
+    try:
```

Comment:
I would remove all of the try/except branches. If anything fails, they will throw their own exceptions.

Reply:
Can I at least leave the `RequestException` case? I would like to see the 400's and 500's at the logs early rather than wait until it breaks all the sync.